die ene facebookpuzzel.py

```diff
@@ -1,24 +1,24 @@
-ops = ["+", "-", "*", "/"]
 
-def func():
+def solve(problem, operations, base=""):
 	"""
 	Een functie om Daphne's breinbrekende facebookpuzzel op te lossen.
 	
 	Het doel van deze puzzel is om een som op te lossen door operatoren in te vullen in de som.
 	
 	Pas op als je computer bijzonder aardappel is, omdat er wel 4^4=256 opties getest worden!
 	"""
```

Comment:
Deze afgrijselijke code heeft verwarrende documentatie over de parameters nodig.

Reply:
Ah yes, goeie. Ik zal proberen wat spellingsfouten en outdated comments toe te voegen.

Reply:
Ooh, en een nutteloze parameter voor een TODO functionaliteit die je wel altijd moet invullen als je de functie wil gebruiken hoewel het momenteel niks doet
"""pk: the private key for the upcoming solve-over-blockchain feature (last updated: 20-8-2012)"""

Reply:
Geniaal